inc/options/class-wpseo-option.php

```diff
@@ -722,144 +700,4 @@ protected function get_switch_key( $key ) {
 
 		return $key;
 	}
-
-	/* *********** DEPRECATED METHODS *********** */
```

Comment:
I'm not sure if we want to remove this header as we might have deprecated methods in the feature.